src/main/webapp/js/omniComment.es6

```diff
@@ -328,18 +398,15 @@ $(document).ready(() => {
         }
     }
 
-    /**
-     * Remove param and its value pair in the given url
-     * Return the url withour param and value pair
-     */
-    function removeParamInUrl(url, param) {
-        let indexOfParam = url.indexOf(`?${param}`);
-        indexOfParam = indexOfParam === -1 ? url.indexOf(`&${param}`) : indexOfParam;
-        const indexOfAndSign = url.indexOf('&', indexOfParam + 1);
-        const urlBeforeParam = url.substr(0, indexOfParam);
-        const urlAfterParamValue = indexOfAndSign === -1 ? '' : url.substr(indexOfAndSign);
-        return urlBeforeParam + urlAfterParamValue;
-    }
```

Comment:
Future PR: the named functions in this gigantic document.ready function should be extracted out unless unless its using enclosing scope's variable.